lib/src/models/comment_impl.dart

```diff
@@ -99,13 +111,17 @@ class MoreComments extends RedditBase {
   }
 
   Future<Comment> _loadComment(String commentId) async {
+    if (_submission is! Submission) {
+      _submission = await _submission.populate();
+    }
+    final submissionCast = _submission as Submission;
```

Comment:
Style Comment: Are you sure we should keep Cast in the name of the variable, it makes it longer and hard to understand what's going on later on in the code when using that variable.

Reply:
Yeah, this is sloppy. I'll fix it. Thanks for calling me on it!